content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -61,22 +61,7 @@ with great caution. The following describes how to quickly experiment with Valid
 
 The following is an example of a ValidatingAdmissionPolicy.
 
-```yaml
-apiVersion: admissionregistration.k8s.io/v1alpha1
-kind: ValidatingAdmissionPolicy
-metadata:
-  name: "demo-policy.example.com"
-spec:
-  failurePolicy: Fail
-  matchConstraints:
-    resourceRules:
-    - apiGroups:   ["apps"]
-      apiVersions: ["v1"]
-      operations:  ["CREATE", "UPDATE"]
-      resources:   ["deployments"]
-  validations:
-    - expression: "object.spec.replicas <= 5"
-```
+{{% code language="yaml" file="validatingadmissionpolicy/basic-example-policy.yaml" %}}
```

Comment:
pre-submit doesn't like something here:

```
3:26:20 PM: Error: Error building site: /opt/build/repo/content/en/docs/reference/access-authn-authz/validating-admission-policy.md:64:1: failed to extract shortcode: template for shortcode code not found
```

@sftim Do you happen to know how to interpret this error message?

Reply:
For now, use `codenew` - we are switching to a new name but the branch you have may not incorporate that change.

Reply:
For faster feedback, [preview locally](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally).